### PR TITLE
Feature/book

### PR DESCRIPTION
**Pull request by frankiannelli**
updated Book, Loan and Member entities

### CODE REVIEW

File: src/library/entities/Book.java [hunk 1]
```diff
@@ -1,95 +1,106 @@
...
 	
 	public String toString() {
 		StringBuilder sb = new StringBuilder();
-		sb.append("Book: ").append(iD).append("\n")
-		  .append("  Title:  ").append(tItLe).append("\n")
-		  .append("  Author: ").append(AuThOr).append("\n")
-		  .append("  CallNo: ").append(CALLNO).append("\n")
-		  .append("  State:  ").append(StAtE);
+		sb.append("Book: ").append(id).append("\n")
+		  .append("  Title:  ").append(titLe).append("\n")
```
Comment (Givo29) [inline src/library/entities/Book.java]: `titLe` here should be `title`

**Review by Givo29 — CHANGES_REQUESTED** [review]

File: src/library/entities/Book.java (hunk 1 above)
Comment (frankiannelli) [reply src/library/entities/Book.java]: nice pickup

**Comment by Givo29** [comment]
Looks good to me

**Review by Givo29 — COMMENTED** [review]
Thanks for the changes, looks good

**Pull request merged**